packages/hardhat-truffle4/src/glob.ts

```diff
@@ -0,0 +1,19 @@
+import { join } from "path";
+
+export async function getSolidityFiles(directory: string): Promise<string[]> {
+  try {
+    const { getAllFilesMatching } = await import(
+      "hardhat/internal/util/fs-utils"
+    );
+
+    return await getAllFilesMatching(directory, (f) => f.endsWith(".sol"));
```

Comment:
I'm not 100% sure about this because it means we can't delete, move or modify `getAllFilesMatching`.

Maybe we should also check that it's not `undefined`?

Reply:
Makes sense. Let's use glob if it's `undefined`.